packages/contracts-bedrock/test/RoundRobinLeaderElection.t.sol

```diff
@@ -63,11 +63,12 @@ contract RoundRobinLeaderElectionTest is Test {
         address notALeader = vm.addr(1234);
         vm.prank(notALeader);
         vm.expectRevert("RoundRobinLeaderElection: submit function must be called by the leader.");
-        leaderContract.submit("hi");
+        LeaderElectionBatchInbox.MetaData[] memory metas = new LeaderElectionBatchInbox.MetaData[](1);
+        leaderContract.submit(metas);
```

Comment:
I'm not sure if this works with the `external` specifier... if it doesn't, I'm not sure it's worth testing this way, given the difference between `external` and `public`.

Reply:
I think it should work.

Reply:
Yes, confirmed. Since the test contract is another contract that makes sense.